eclass/llvm-utils.eclass

```diff
@@ -150,4 +150,19 @@ llvm_prepend_path() {
 	export PATH=${new_path[*]}
 }
 
+# @FUNCTION: llvm_cmake_usex_musl
+# @DESCRIPTION:
+# Determine whether the given LLVM project should be built with musl
+# support. That should be the case if the CTARGET (or CHOST) is a musl
+# environment.
+#
+# If musl should be used, echo "yes", otherwise echo "no".
+llvm_cmake_usex_musl() {
+	if [[ "${CTARGET:-${CHOST}}" == *-*-*-musl* ]]; then
+		echo "yes"
+	else
+		echo "no"
```

Comment:
```suggestion
		echo "ON"
	else
		echo "OFF"
```

These are the canonical values. `usex` is using yes/no because historical reasons (read: someone designed the whole thing around a single package using yes/no, sigh), and CMake just happens to support them.

Reply:
Done